### PR TITLE
drizzle plugin

### PR DESCRIPTION
**Pull request by jgoux**
Docs: https://orm.drizzle.team/kit-docs/overview

I also added `playwright-ct` to the `schema.json` file.

### CODE REVIEW

File: fixtures/plugins/drizzle/drizzle.config.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import type { Config } from "drizzle-kit";
+
+export default {
+  schema: "./src/schema.ts",
```
Comment (jgoux) [inline fixtures/plugins/drizzle/drizzle.config.ts:4]: I think ideally we should mark this file as entrypoint as well, but not sure how to do that.

Comment (webpro) [reply fixtures/plugins/drizzle/drizzle.config.ts:4]: Inside `findDependencies` you can `load` the `configFilePath` (first argument) and then return the value `config.schema` in the array of dependencies.

Comment (jgoux) [reply fixtures/plugins/drizzle/drizzle.config.ts:4]: Thanks, on it. Which lib should I use to expand/process globs?

Comment (webpro) [reply fixtures/plugins/drizzle/drizzle.config.ts:4]: That should be mostly handled, the plugin can literally return e.g. `['.src/schema.ts']` and Knip will figure it out (e.g. abs/rel paths are entries, the rest is package names).

The first argument for `findDependencies` is `configFilePath` and that should be an absolute path already.

Comment (webpro) [reply fixtures/plugins/drizzle/drizzle.config.ts:4]: Here's a similar example: https://github.com/webpro/knip/blob/main/src/plugins/commitizen/index.ts#L25-L26

Comment (jgoux) [reply fixtures/plugins/drizzle/drizzle.config.ts]: `drizzle` accepts globs as well, I tried returning them but it didn't work, so I now process the paths with globby before returning the array, it did the trick. 👍

**Comment by jgoux** [comment]
It should be good for a review. ✅

**Comment by webpro** [comment]
Thanks, Julien! 🙏

**Comment by webpro** [comment]
:rocket: _This pull request is included in v2.28.0. See [Release 2.28.0](https://github.com/webpro/knip/releases/tag/2.28.0) for release notes._

**Pull request merged**